go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/openmeterio/openmeter
 
 go 1.22.0
 
+replace github.com/getkin/kin-openapi => github.com/getkin/kin-openapi v0.123.0
```

Comment:
This is not inherited in case somebody uses our API directly from the repo. Let's make sure that the client is working without this replace (I think it only affects the code generation).

Reply:
i'm not sure what you mean by " Let's make sure that the client is working without this replace". The bug only affects the code generation.

also https://github.com/openmeterio/openmeter/pull/935

Reply:
Thank you for confirming, that's what I was asking for ;-)